Models/ProtoModel.cpp

```diff
@@ -89,14 +88,20 @@ void ProtoModel::DisplayConfig::SetFieldDefaultIcon(const FieldDescriptor *field
 
 void ProtoModel::DisplayConfig::SetFieldHeaderIcon(const std::string &message, const FieldPath &field_path,
                                                    const QString &icon_name) {
-  std::string field = message;
-  message_display_configs_[message].icon_field = field_path;
```

Comment:
Pretty sure this wasn't supposed to be in this method? The `SetMessageIconIdField` already covers its behavior.

Reply:
I see the other one is actually specifying a lambda lookup, so maybe that's why.
I think there's more permutations of these functions that can be added later actually.
I put them back for now and decided to punt: 0c935917fcf42f6990fa3730ba69e08840d125a1